### PR TITLE
Reformat "How to build" Readme

### PR DESCRIPTION
**Pull request by ldunnplaymint**

### CODE REVIEW

**Comment by 5p0rt5BEArD** [comment]
I like the tutorial features. Perhaps remove the key column to give more space.

**Comment by 5p0rt5BEArD** [comment]
I'd like the existing tables under examples to follow the same new format and be brought to the top of the page.

As per #1226 

Then the rest can be hidden behind a collapsible section like you suggested.

**Comment by ldunnplaymint** [comment]
Do you mean the example buildings?

**Comment by ldunnplaymint** [comment]
I have rearranged it but not made any edits to the examples table. If you are happy with that format, I will update the content tomorrow.

**Comment by 5p0rt5BEArD** [comment]
> Do you mean the example buildings?

There used to be example-plugins and maps.

Now there is just maps. Some example-plugins have become maps some have disappeared.

Every folder in the new set of maps folders should have a corresponding entry in a table. It might be a single table or it could be split into

Tutorials
Examples 
Full-Games

**Comment by ldunnplaymint** [comment]
I have pulled all the tables up and roughly organised them. Again, no content changes just yet, want to make sure it's close to what you are describing.

File: tutorial/README.md [hunk 1]
```diff
@@ -3,16 +3,45 @@
...
+| tonk        |<img src="images/map-tonk.png" width="200">     | Game by [Tonk](https://github.com/tonk-gg/ds-extensions-debuggus). Uses: Connection to external server; extensive UI take over; item UI plugin; Unit model swaps |
+| hexcraft    |<img src="images/map-hexcraft.png" width="200"> | Game from [1kx](https://github.com/1kx-network/hexcraft). Uses: Team allocation; Building contract code linked; Restricted crafting; Restricted building; Unit model swaps |
+| labyrinth   |<img src="images/map-labyrinth.png" width="200">| Game from [RockawayX](https://github.com/rockawayx-labs/ds-dx). Uses: Doors; Password hashing; Combat Stats; Item checking; Quests; Map reset|
+| duck-burger |<img src="images/map-tonk.png" width="200">     | Game by [Playmint] (https://www.playmint.com). Learn how to make DVB using [Tutorial 5](../contracts/src/maps/tutorial-room-5/)|
+
+### Please note:
+Tonk attack requires Tonk services to be running: Run with docker using the `tonk` profile: `docker compose --profile tonk up`
+
+<details>
+  <summary> References </summary>summary>
```
Comment (chrisfarms) [inline tutorial/README.md:43]: something not quite right here...

```suggestion
  <summary> References </summary>
```

File: tutorial/README.md [hunk 2]
```diff
@@ -3,16 +3,45 @@
...
+
+## Existing games
+__[All example games can be found here](../contracts/src/maps)__ 
+
+| Folder      | Tile Fabricator View                           | Description       |
+|:------------|:----------------------------------------------:|:------------------|
+| tonk        |<img src="images/map-tonk.png" width="200">     | Game by [Tonk](https://github.com/tonk-gg/ds-extensions-debuggus). Uses: Connection to external server; extensive UI take over; item UI plugin; Unit model swaps |
+| hexcraft    |<img src="images/map-hexcraft.png" width="200"> | Game from [1kx](https://github.com/1kx-network/hexcraft). Uses: Team allocation; Building contract code linked; Restricted crafting; Restricted building; Unit model swaps |
+| labyrinth   |<img src="images/map-labyrinth.png" width="200">| Game from [RockawayX](https://github.com/rockawayx-labs/ds-dx). Uses: Doors; Password hashing; Combat Stats; Item checking; Quests; Map reset|
+| duck-burger |<img src="images/map-tonk.png" width="200">     | Game by [Playmint] (https://www.playmint.com). Learn how to make DVB using [Tutorial 5](../contracts/src/maps/tutorial-room-5/)|
```
Comment (chrisfarms) [inline tutorial/README.md:37]: broken link

```suggestion
| duck-burger |<img src="images/map-tonk.png" width="200">     | Game by [Playmint](https://www.playmint.com). Learn how to make DVB using [Tutorial 5](../contracts/src/maps/tutorial-room-5/)|
```

File: tutorial/README.md [hunk 3]
```diff
@@ -3,16 +3,45 @@
...
+
+### Please note:
+Tonk attack requires Tonk services to be running: Run with docker using the `tonk` profile: `docker compose --profile tonk up`
+
+<details>
```
Comment (chrisfarms) [inline tutorial/README.md:42]: personally I think there is way too much hidden behind this expandy section.

Probably better to give all this it's own `reference.md` file, and link to it from both this file and the top level readme.

Comment (ldunnplaymint) [reply tutorial/README.md]: Done this

File: README.md [hunk 4]
```diff
@@ -161,3 +161,6 @@ NUM_ARENAS=4 make contracts/src/maps/performance-test
 
 You can then either start locally via `MAP=performance-test make dev` or manually `ds apply -R -f contracts/src/maps/performance-test`
 </details>
+
+# References
+__[For more information, check the reference page](/tutorial/REFERENCE.md)__
```
Comment (chrisfarms) [inline README.md:166]: this link could go within the "building on downstream" section imo ... but I don't feel strongly

**Review by chrisfarms — APPROVED** [review]

**Comment by ldunnplaymint** [comment]
I think Iain will have opinions so will just bank it for now

**Pull request merged**